off-chain/src/Network/Types.purs

```diff
@@ -26,17 +32,61 @@ type MiliSeconds = Int
 type Time = Int
 type MutableInt = Int
 
-newtype IncludeActionError = RejectedServerBussy LeaderServerStateInfo
+data IncludeActionError
+  = RejectedServerBussy LeaderServerStateInfo
+  | IAOtherError String
```

Comment:
At first, I thought that the rejection is the only error I can think of, but maybe we can have an error while decoding a response, are there other cases apart from those?

Reply:
I thought we were to distinguish the two kinds of errors, the ones made by Seath and those made by payload.

Reply:
yes, this approach with` try` and `data IncludeActionError = RejectedServerBussy` is what I thought.

Reply:
I will be changing occurrences while working on related parts. Feel free to change them also where needed.